### ISSUE TITLE
[issue name]

### ISSUE THREAD

**Issue by Jinxsyns** [iron-fish/ironfish#5323]
### What happened?

I have a transaction that I used the "fast" transaction fee that has been pending in the node app the submitted sequence was 705712 when the current height is 705963 and when trying to look it up in the block explorer it says "This transaction either does not exist or hasn’t made it to the explorer yet." which has never happened this long before. It's been roughly 3 hours that it has been pending. is there anyway to retry the transaction? or get it unstuck?

### Version

3.0.8

### Debug output

```shell
It is the gui node I don't know how to get this information I tried enabling telemetry just now idk if that will do anything.
```

### Relevant log output

```shell
Same as with the debug output i'm unsure how to get that info
```

**Comment by Jinxsyns** [comment]
Well it just expired so let me give it another shot I will close this if it works

**Comment by Jinxsyns** [comment]
Yup it's stuck again for some reason I absolutely cannot seem to get the transaction to work sending it to tradeogre which is strange because I did it about 26 days ago with no problems.

**Comment by NullSoldier** [comment]
When you did it again, can you find it in the block explorer? Or is that second attempt also not found in the block explorer?

**Comment by NullSoldier** [comment]
Also, can you post the information you can see in `Your Node` -> `Overview` in the orange card? A screenshot is also fine. This card does not have any private information on it.

**Comment by Jinxsyns** [comment]
I can when i get home later tonight sorry i am at work for another 6 hours.

**Comment by Jinxsyns** [comment]
> When you did it again, can you find it in the block explorer? Or is that second attempt also not found in the block explorer?
No I couldn't either time.

 
> Also, can you post the information you can see in `Your Node` -> `Overview` in the orange card? A screenshot is also fine. This card does not have any private information on it.

![Screenshot 2024-08-21 031948](https://github.com/user-attachments/assets/80aacd70-cc60-4ba0-9226-01047f54387a)

looking at this deeper I did just notice that my node wasn't fully synced could that be the problem? I swapped frommmainnet to testnet which reset the sync to the right head sequence but it was stuck at 2 blocks behind the current block height when i finished initiating the transaction. (transaction initiated somewhere between block 706536 and 706538) To clarify just in case the node is stuck at 706536 and the height at the time of initiating the transaction was 706538
